src/bp/core/database/tables/bot-specific/logs.ts

```diff
@@ -7,7 +7,7 @@ export class LogsTable extends Table {
     let created = false
     await this.knex.createTableIfNotExists(this.name, table => {
       table.string('botId').nullable()
-      table.string('timestamp')
+      table.timestamp('timestamp')
```

Comment:
Since you're changing the column type here, I'd check in your migration that the column type is not a "timestamp". Otherwise the migration will run even though its not necessary.